README.md

```diff
@@ -116,13 +116,17 @@ instance.interceptors.request.use((request) => {
 
 #### Enable config list
 
```

Comment:
see https://github.com/ClintEsteMadera/axios-logger/tree/add-support-for-other-loggers-than-just-console-log#enable-config-list to appreciate how this will look like. I think it's a tad cleaner this way...

Reply:
Looks good table!